prow/ANNOUNCEMENTS.md

```diff
@@ -1,6 +1,9 @@
 # Announcements
 
 New features added to each component:
+
+ - *January, 5, 2018*`blunderbluss` plugin now provides a new command, `/auto-cc`,
+   that will trigger assigning reviewers manually.
```

Comment:
Err, automatically?

Reply:
I had to think about this for a few seconds. :grin:

`blunderbluss` request reviews _automatically_ when a PR is created. What the command does is _manually_ trigger the same effect.

The following should be clearer:

> `blunderbluss` plugin now provides a new command, `/auto-cc`, that will trigger automatic review requests.